src/renderer/status-pill/pill.css

```
@@ -0,0 +1,647 @@
+@import 'tailwindcss';
+
+@font-face {
+  font-family: 'Geist';
```

Stylelint (17.14.0)

[error] Expected no quotes around "Geist" (font-family-name-quotes)

(font-family-name-quotes)